_templates/component/new/default-entry.ejs.t

```diff
@@ -0,0 +1,11 @@
+---
+to: src/components/<%= h.inflection.dasherize(component_name) %>/index.jsx
+---
+
+import './_index.scss';
```

Comment:
This isn't needed - we should not be loading styles via JS

Reply:
oh ok. updated